frontend/src/pages/AccountInfo.tsx

```diff
@@ -48,37 +48,35 @@ export default class AccountInfo extends React.Component<
 
 	render() {
 		return (
-			<div>
+			<div className="registerBody">
 				<div className="h1">Account Information</div>
-				<div className="input-wrapper">
-					<div className="input">
-						<Group>
-							<Label>First Name:</Label>
-							<Input
-								type="text"
-								placeholder="This should be acc first name."
-								value={this.state.firstName}
-								disabled
-							/>
+				<div className="input">
+					<Group>
+						<Label>First Name:</Label>
+						<Input
+							type="text"
+							placeholder="This should be acc first name."
```

Comment:
@vong98  Can you move these to a separate file called `PageContent.ts` it should return a simple object with all of these static strings. example below.
``` typescript
# PageContent.ts
export default {
	firstName: {
		label: "First name:",
		placeholder: "some placeholder string"
  },

	lastName: {
		label: "Last name:",
		placeholder: "some placeholder string"
	}
};
````

This means if we want to change aesthetic things like the content we only need to edit the corresponding string in one file and not effect component src.

In my opinion, we should start this patter earlier rather than latter.

Reply:
done! not sure if it's standard practice to resolve this conversation or not.